src/gdcdictionary/schemas/treatment.yaml

```diff
@@ -6764,309 +6765,312 @@ properties:
-      - Lymph Node(s) Pelvic
-      - Lymph Node(s) Popliteal
-      - Lymph Node(s) Regional
-      - Lymph Node(s) Retroperitoneal
-      - Lymph Node(s) Scalene
-      - Lymph Node(s) Splenic
-      - Lymph Node(s) Subclavicular
-      - Lymph Node(s) Submandibular
-      - Lymph Node(s) Supraclavicular
-      - Mandible
-      - Mantle
-      - Maxilla
-      - Mediastinal Soft Tissue
-      - Mediastinum
-      - Mesentery
-      - Mesothelium
-      - Middle Finger
-      - Mitochondria
-      - Muscle
-      - Nails
-      - Nasal Cavity
-      - Nasal Soft Tissue
-      - Nasopharynx
-      - Neck
-      - Nerve
-      - Nerve(s) Cranial
-      - Occipital Cortex
-      - Ocular Orbits
-      - Omentum
-      - Oral Cavity
-      - Oral Cavity - Mucosa Only
-      - Oropharynx
-      - Ovary
-      - Palate
-      - Pancreas
-      - Parametrium
-      - Paraspinal Ganglion
-      - Parathyroid
-      - Parotid Gland
-      - Patella
-      - Pelvis
-      - Penis
-      - Pericardium
-      - Periorbital Soft Tissue
-      - Peritoneal Cavity
-      - Peritoneum
-      - Pharynx
-      - Pineal
-      - Pineal Gland
-      - Pituitary Gland
-      - Placenta
-      - Pleura
-      - Popliteal Fossa
-      - Pouch
-      - Primary Tumor Field
-      - Primary tumor site
-      - Prostate
-      - Prostate Bed
-      - Prostate, Seminal Vesicles and Lymph Nodes
-      - Pylorus
-      - Rectosigmoid Junction
-      - Rectum
-      - Regional Site
-      - Retina
-      - Retro-Orbital Region
-      - Retroperitoneum
-      - Rib
-      - Ring Finger
-      - Round Ligament
-      - Sacrum
-      - Salivary Gland
-      - Scalp
-      - Scapula
-      - Sciatic Nerve
-      - Scrotum
-      - Seminal Vesicle
-      - Shoulder
-      - Sigmoid Colon
-      - Sinus
-      - Sinus(es), Maxillary
-      - Skeletal Muscle
-      - Skin
-      - Skin, lower extremity, local
-      - Skin, total
-      - Skin, trunk, local
-      - Skin, upper extremity, local
-      - Skull
-      - Small Bowel
-      - Small Bowel - Mucosa Only
-      - Small Finger
-      - Soft Tissue
-      - Spinal Column
-      - Spinal Cord
-      - Spine
-      - Spleen
-      - Splenic Flexure
-      - Sternum
-      - Stomach
-      - Stomach - Mucosa Only
-      - Subcutaneous Tissue
-      - Synovium
-      - Temporal Cortex
-      - Tendon
-      - Testis
-      - Thigh
-      - Thoracic Spine
-      - Thorax
-      - Throat
-      - Thumb
-      - Thymus
-      - Thyroid
-      - Tibia
-      - Tongue
-      - Tonsil
-      - Tonsil (Pharyngeal)
-      - Trachea / Major Bronchi
-      - Transverse Colon
-      - Trunk
-      - Umbilical Cord
-      - Ureter
-      - Urethra
-      - Urinary Tract
-      - Uterus
-      - Uvula
-      - Vagina
-      - Vas Deferens
-      - Vein
-      - Venous
-      - Vertebra
-      - Vulva
-      - White Blood Cells
-      - Wrist
-      - Other
-      - Unknown
-      - Not Reported
+    type: array
```

Comment:
Did this property's type change from a single enum to an array of enums?  if so, that's a breaking change.

Reply:
Good catch, will update node and ticket to deprecate/migrate singular to an array plurality (treatment_anatomic_sites)